LMChatCore_iOS/LMChatCore_iOS/Source/Module/LMAIChatBotChat/LMAIChatBotInitiationViewModel.swift

```diff
@@ -49,13 +49,25 @@ public class LMAIChatBotChatViewModel: LMChatBaseViewModel {
     
     /// Factory method to create an instance of `LMAIChatBotViewController`
     /// wired up with an `LMAIChatBotChatViewModel`.
-    public static func createModule() throws -> LMChatAIBotInitiationViewController {
+    public static func createModule() throws -> LMViewController {
         guard LMChatCore.isInitialized else {
             throw LMChatError.chatNotInitialized
         }
-        let viewController = LMCoreComponents.shared.aiChatBotIntiationScreen.init()
-        viewController.viewModel = LMAIChatBotChatViewModel(viewController)
-        return viewController
+        
+        
+        if let existingChatroomId = LMSharedPreferences.getString(forKey: "chatroomIdWithAIChatbot") {
+            // If we have an existing chatroom ID, navigate directly to the chat screen
+            return try LMChatMessageListViewModel.createModule(
+                withChatroomId: existingChatroomId,
+                conversationId: nil
+            )
+        } else {
+            // If no existing chatroom, show the initiation screen
+            let viewController = LMCoreComponents.shared.aiChatBotIntiationScreen.init()
+            viewController.viewModel = LMAIChatBotChatViewModel(viewController)
```

Comment:
the name of the viewmodel should be changed here as well, it should be similar to 
```suggestion
            viewController.viewModel = LMChatAIBotInitiationViewModel(viewController)
```

Reply:
done